tests/crypto/README.rst

```diff
@@ -0,0 +1,306 @@
+| Hash               | SHA256      | `NIST - SHA`_                                                              | `NIST - SHA`_                                                              |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | SHA512      |                                                                            | `NIST - SHA`_                                                              |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HMAC               | HMAC SHA256 | `NIST - HMAC`_                                                             | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HMAC SHA512 |                                                                            | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HKDF               | HKDF SHA256 | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HKDF SHA512 |                                                                            | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| EC-JPAKE           | secp256r1   | `J-PAKE: Password-Authenticated Key Exchange by Juggling`_                 | Custom                                                                     |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF9160DK|
+  * |nRF52840DK|
+  * |nRF52DK|
+  * |nRF51DK|
+
+.. note::
+   Nordic devices with smaller flash capacity, such as nRF51, nRF52810, or nRF52811 might not fit the full test suite.
```

Comment:
```suggestion
   The full test suite might not fit on Nordic devices with smaller flash capacity, such as nRF51, nRF52810, or nRF52811.
```
Not sure I like that wording, but "fit" must be used the other way around. Alternatively, you could write something like "Nordic devices such as ... cannot run the full test suite because of limited flash capacity".

Reply:
@ru-fu: Your suggestion is better

Reply:
done